cmd/keymasterd/main.go

```diff
@@ -1151,6 +1156,18 @@ func (state *RuntimeState) startVIPPush(cookieVal string, username string) error
 	return nil
 }
 
+func getLoginDestination(r *http.Request) string {
+	loginDestination := profilePath
+	if r.Form.Get("login_destination") != "" {
+		inboundLoginDestination := r.Form.Get("login_destination")
+		if strings.HasPrefix(inboundLoginDestination, "/") &&
+			!strings.HasPrefix(inboundLoginDestination, "//") {
```

Comment:
Would be helpful to put a few words in a comment why we don't want the `//` for future reference.